rules/rules-reviewed/azure/azure-java-version.windup.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<ruleset id="azure-java-version"
+         xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Check non-LTS Java versions
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot"/>
+        <sourceTechnology id="eap" versionRange="[7,8)" />
+        <targetTechnology id="azure-spring-apps"/>
+        <targetTechnology id="azure-appservice"/>
+        <targetTechnology id="azure-aks"/>
+        <targetTechnology id="azure-container-apps"/>
+        <tag>version</tag>
+    </metadata>
+    <rules>
+        <rule id="azure-java-version-01000">
+            <when>
+               <xmlfile matches="//m:java.version[windup:matches(text(), '{v}')]" in="pom.xml" as="result">
+                        <namespace prefix="m" uri="http://maven.apache.org/POM/4.0.0"/>
```

Comment:
Continuin with my comments, in this case, this is correct, since the POM files in the test folders have namespace information, although [it is possible to find POM files without namespace information that build correctly](https://stackoverflow.com/questions/25996084/pom-xml-namespace-necessary). If you want to capture those edge cases, you will have to `<or>` this condition with an additional one _without_ the `namespace` element.